test/e2e/testing-manifests/storage-csi/mock/csi-mock-proxy.yaml

```diff
@@ -0,0 +1,90 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-mockplugin
+spec:
+  selector:
+    matchLabels:
+      app: csi-mockplugin
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: csi-mockplugin
+    spec:
+      serviceAccountName: csi-mock
+      containers:
+        - name: csi-provisioner
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v1.6.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+            # Topology support is needed for the pod rescheduling test
+            # ("storage capacity" in csi_mock_volume.go).
+            - "--feature-gates=Topology=true"
+            - "-v=5"
+            - "--timeout=1m"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+        - name: driver-registrar
+          # TODO: replace with official image that includes https://github.com/kubernetes-csi/node-driver-registrar/pull/124
+          image: pohly/csi-node-driver-registrar:v1.3.0
+          args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/csi-mock/csi.sock
+            - --timeout=1m
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+        - name: mock
+          # TODO: replace with official image which includes https://github.com/kubernetes-csi/csi-test/pull/314
+          image: pohly/mock-driver:2020-12-04-1
+          args:
+            - -v=5
+            - -proxy-endpoint=tcp://:9000
+          env:
+            - name: CSI_ENDPOINT
+              value: /csi/csi.sock
+          ports:
+            - containerPort: 9000
+              name: socat
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+        # The busybox container is needed for running shell commands which
+        # test for directories or create them.
+        - name: busybox
```

Comment:
If I understood it correctly, this container needs to privileged, because it's going to create directories in `/var/lib/kubelet/pods/...`. On machines with SELinux, only privileged containers may modify it.

Reply:
Correct, will fix this.

Reply:
Done.

Reply:
I still see it as unprivileged

Reply:
Where the heck did my change go? Ah, into the wrong container (external-provisioner above). Fixed.